homeassistant/components/home_connect/switch.py

```diff
@@ -110,41 +106,48 @@ async def async_setup_entry(
 ) -> None:
     """Set up the Home Connect switch."""
 
-    def get_entities() -> list[SwitchEntity]:
+    def get_entities_for_appliance(
```

Comment:
We don't need this function since it's only called once. We can inline the code.